packages/spright-components/src/chat/input/tests/chat-input.spec.ts

```diff
@@ -176,6 +170,17 @@ describe('ChatInput', () => {
             expect(spy).toHaveBeenCalled();
         });
 
+        it('clears the input contents, sets focus, and disables the button', () => {
```

Comment:
Could you add a test that calls `focus()` on the chat-input and verifies that the text area receives focus? I think it will be common for apps to want to start out in that state. Previously they could call `resetInput()` to guarantee it but now calling `focus()` would be the way to do it. (They might have tried calling `focus()` before too and it would work, but we didn't have test coverage for it).